contents/incentivization.rst

```diff
@@ -0,0 +1,134 @@
+
+Every project with cryptoeconomic elements defines its own rules, protocols and incentives system to reward users, which reflect the different needs and operating principles of the specific network.
+
+For Swarm to properly function as a decentralized p2p storage and communication infrastructure, on very basic terms its peers need to:
+ * contribute bandwidth for incoming and outgoing requests
+ * provide storage for users to upload and retrieve data
+ * forward incoming requests to peers who can fulfill them if they can not serve the request themselves
+
+Swarm introduces its own incentives system for ensuring correct network behavior by rewarding nodes for serving these functions. The general swarm docs specify different functional layers of Swarm, which roughly correspond to its implementation roadmap:
+ * accounting system
+ * file insurance
+ * litigation
+
+
+At this point we only document the accounting system. The other parts will be documented in the future, although they are specified partly already in the documentation (The Book Of Swarm). This documentation also only is meant as an entry point for understanding and operating Swarm. For a thorough and detailed documentation the Book of Swarm should be consulted.
+
+Accounting
+==========
+At the core of Swarm's incentive system is the SWarm Accounting Protocol, or SWAP. It is a very simple protocol which relies on a tit-for-tat p2p accounting model:
+ * a node gets rewarded for serving resources
+ * a node gets charged for requesting resources
+
+Every peer maintains a local database on its own file system which contains accounting information of all peers with which it exchanged data. 
+
+.. note::
+
+   SWAP excels at its simplicity. It is not the goal and would completely defeat the purpose of the system to introduce bullet-proof consistency and auditing - Swarm is not a blockchain in its own terms.`
+
+
+.. important::
+   The Swarm node operator is responsible for maintenance of the local accounting database (a LevelDB instance). Tampering with it, altering its contents or removing it altogether may result in the node becoming inoperable.
+
+If a node requests data from a peer, when the peer delivers the data, the peer credits an amount equivalent to the price of that message in its local database, at the index represting the peer. The node, when it receives the message (data) from the peer, debits itself in its local database the same amount at the index of the peer. 
```

Comment:
I do think we should because this is a documentation mainly to get started and understand the basics. Exactly this kind of information is nowhere in any documentation in fact, so I believe it should be added here.

Reply:
i agree. but i still would present the concept first, then the technical aspect of it later.

not a blocker though, we can agree to disagree